### ISSUE TITLE
Accessibility: Common: [MAS4.2.1]: Screen reader is not announcing the selected state of the links in top nav.

### ISSUE THREAD

**Issue by tylerbutler** [microsoft/FluidFramework#3813]
**Build details:**

**OS Version:** 2004(20221.1000)

**Edge Dev Version**: 87.0.654.0 (Official build) dev (64-bit)
 

**Pre-requisite:**

Turn on narrator.  
  

**Repro Steps:**

1\. Navigate to 'Fluid Framework' link in the top nav.

2.Now listen to the screen reader announcement.

  
**Actual Result**  
Screen reader is not announcing the selected state of the links in top nav. It is only announcing like "link,fluid framework".

  
**Expected Result:**  
Screen reader should announce as ‘Currently Selected’ for currently selected links in the top nav.  
  
**Note**:

This issue is observed in all the given links as well-

 https://fluidframework.com/docs/
 https://fluidframework.com/apis/
https://fluidframework.com/community/
https://fluidframework.com/playground/?path=/docs/welcome--page
https://fluidframework.com/docs/get-started/quick-start/
https://fluidframework.com/docs/get-started/tutorial/
https://fluidframework.com/docs/get-started/examples/

  
**User Impact:**  

Users with visual impairment who are dependent on screen readers will not get to know which link is currently selected in the top nav.

  

**MAS Reference:**

**MAS 4.2.1 - Object Information**
